_people/daniel_morris.md

```diff
@@ -4,5 +4,5 @@ headshot: cxWKwjn
 gender: male
 submitted: false
 course:
-  - Physics MSc
+  - MSc Physics
```

Comment:
What’s the difference between an MSc and an MSci?

Reply:
Specifically given Dan and David graduated close to one another and whether UoN offers a MSc Physics

Reply:
Seems like you do a BSc but an MSci